### ISSUE TITLE
Model where_clause should be a class method not an instance method

### ISSUE THREAD

**Issue by kauberry** [EMSL-MSC/pacifica-metadata#85]
I think it makes more sense to have the where_clause generator in each of the model classes be a class method rather than an instance method.

Asking for a set of records of a given type is not reliant on the instance in any way that I can think of.

**Comment by dmlb2000** [comment]
I agree, I think there's more than just the where_clause that would be class methods... Looking at `metadata.orm.base` we probably can make class methods out of a lot of the instance methods there.
